calico/networking/configuring/qos-controls.mdx

```diff
@@ -0,0 +1,117 @@
+---
+description: Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections of Calico workloads.
+---
+
+# Configure QoS Controls
+
+## Big picture
+
+Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections used by Calico workloads in order to prevent them from overusing or overloading network resources.
+
+## Value
+
+With QoS controls, $[prodname] enforces network resource limits (bandwidth, packet rate, connections) for Kubernetes pods and OpenStack VMs, ensuring fair resource allocation and preventing performance degradation for other workloads.
+
+## Concepts
+
+### Quality of Service Controls
+
+**Quality of Service** (**QoS**), in the field of computer networking, refers to traffic prioritization and resource reservation control mechanisms. It may either mean guaranteeing minimum levels of networking resources, or limiting the maximum levels of usage of networking resources in order to prevent a small number of users from excessive resource consumption, stopping others from using their fair share. For practical purposes, the latter will be considered as $[prodname] implements QoS Controls by limiting maximum levels of network resources usage.
+
+$[prodname] implements three types of QoS Controls, which can be configured using annotations on the Kubernetes pods for either ingress (traffic coming into the pod), egress (traffic going out of the pod), or both:
+1. Bandwidth: limiting the bit rate of traffic into/out of the pod
+2. Packet rate: limiting the number of packets per second that can be sent or received by the pod
+3. Number of established connections: limiting the absolute number of connections that the pod may initiate or accept
+
+### Annotations
+
+On Kubernetes, $[prodname] uses annotations on pods for the configuration of QoS controls. You can annotate pods in any of these ways:
+- pods can be directly annotated with `kubectl annotate`;
+- they can be created with the annotations in their metadata;
+- if they're part of a deployment or daemonset, they can have the annotations in the metadata of their template spec.
+
+The values of the QoS controls annotations may contain suffixes, such as `k`, `M`, `G`, `T`, `P`, or `E` to represent large numbers. Binary suffixes are also supported, for example `1Ki` represents "1024", and so on. For more details, see the [Quantity type of the resource package](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity).
```

Comment:
```suggestion
The values of the QoS controls annotations may contain suffixes, such as `k`, `M`, `G`, `T`, `P`, or `E` to represent large numbers. Binary suffixes are also supported, for example `1Ki` represents "1024".
```

I clicked on the link, it got me to a pretty complex section, confusing.

Reply:
Thanks! Done.